### PR TITLE
feat(resources): add overview for gtfs - operational data standard

### LINKED ISSUE #300: New resource request: ODS one pager
### Approved?

Yes

### Title

Overview: Operational Data Standard

### Is the document correctly named and formatted?

- [X] Did you start the filename with `Cal-ITP`?
- [X] Did you remove all spaces between words in the filename? The only acceptable characters in the filename are letters, numbers, `-`, `.` and `_`.
- [X] Did you ensure there are no revision dates or numbers in the filename? Limit usage of revision numbers or dates unless commonly understood and expected by the audience. Avoid `report-june-2024.pdf`, `file-v1.2.pdf`. Allowed `GTFS-guidlines-v3.pdf`, as version 3 is an official version number.
- [X] Did you run this PDF through Adobe Acrobat's Accessibility Checker, and ensure the PDF passes all accessibility checks?

### Asset URL

https://drive.google.com/file/d/1U8DKyqj9Nx5J0rpgVlHMBsctEm0I6IIO/view?usp=sharing

### Category

Fact sheets & overviews

### Tag

GTFS

### Date - Year

2023

### Date - Month

11

### PR DESCRIPTION
**Pull request by machikoyasuda**
closes #300 

image
image

### CODE REVIEW

File: src/_resources/fact-sheet-ods-overview.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+date: "2023-11-08T17:00:00-07:00"
+title: "Overview: Operational Data Standard"
+asset: https://drive.google.com/file/d/1U8DKyqj9Nx5J0rpgVlHMBsctEm0I6IIO/view
```
Comment (angela-tran) [inline src/_resources/fact-sheet-ods-overview.yml:4]: The latest version of this asset was uploaded in this comment: https://github.com/cal-itp/calitp.org/issues/300#issuecomment-1932527083

**Review by angela-tran — CHANGES_REQUESTED** [review]

File: src/_resources/fact-sheet-ods-overview.yml (hunk 1 above)
Comment (machikoyasuda) [reply src/_resources/fact-sheet-ods-overview.yml]: Updated to new link. The hand is correct now 😆 
image

**Comment by machikoyasuda** [comment]
@thekaveman this is ready for review

**Review by thekaveman — CHANGES_REQUESTED** [review]
This is really minor (I think), but the [latest version uploaded by Sarah](https://github.com/cal-itp/calitp.org/issues/300#issuecomment-1932527083) had removed the page number at the bottom:

image

I'm still seeing it on this linked version in the preview site:

image

**Comment by machikoyasuda** [comment]
Ahh good catch! I updated the PDF so the link now shows file w/o page number.

https://drive.google.com/file/d/1TdMKHTqNLv6HzR2_zSagy_QKzJT9Nwfe/view

image

**Review by thekaveman — APPROVED** [review]
Looks great, thank you!

**Pull request merged**